Adyen/UI/Form/Items/Toggle/FormToggleItemView.swift

```diff
@@ -13,19 +14,52 @@ public final class FormToggleItemView: FormValueItemView<Bool, FormToggleItemSty
     /// Initializes the switch item view.
     ///
     /// - Parameter item: The item represented by the view.
-    public required init(item: FormToggleItem) {
-        super.init(item: item)
-        
-        showsSeparator = false
-        
-        isAccessibilityElement = true
-        accessibilityLabel = item.title
-        accessibilityTraits = switchControl.accessibilityTraits
-        accessibilityValue = switchControl.accessibilityValue
-        
-        addSubview(stackView)
-        stackView.adyen.anchor(inside: self.layoutMarginsGuide)
-    }
+	public required init(item: FormToggleItem) {
+		super.init(item: item)
+		titleLabel = {
+			let titleLabel = TextCheckingLabel(style: item.style.title)
+		 
+			if let string = item.title, string == localizedString(.payment_authorize, nil) {
+				titleLabel.touchHandler = { textCheckingResult in
+					guard let url = textCheckingResult.url else {
+						if #available(iOS 14.0, *) {
+							Logger().error("No url for label: \(titleLabel)")
+						}
+						
+						return
+					}
+					
+					UIApplication.shared.open(url)
+				}
+				titleLabel.attributedText = {
+					let mas = NSMutableAttributedString(string: string)
+					let range = mas.mutableString.range(of: localizedString(.spin_terms, nil))
+					
+					mas.addAttributes([.link: URL(string: "https://www.spin.pm/legal?tab=terms#terms-tab")!, .underlineStyle: NSUnderlineStyle.single.rawValue], range: range)
```

Comment:
I probably lack some context around this project and its future (who owns it, etc) but it seems odd to me that we have spin-specific things in a fork of a 3rd party SDK.

Maybe this would be cleaner if we provided a hook to configure this externally?

What is the long term vision for this forked project?

Reply:
I see this fork as an simple means to enable spin customizations on top of Adyen's official SDK. We current do this for both their Android and iOS which serve as dependencies to our Adyen react native library.

I did not put a whole lot of thoughts into long term vision. Other than keeping our changes minimal as they will need to be frequently rebased to receive updates from the official upstream.

Reply:
yeah this makes me pretty uneasy, I'm not familiar with the structure of this library, or Adyen's willingness to pull in upstream changes, but I would prefer something that we could possibly contribute upstream so that we stay current with their releases.

Reply:
I think we should have a story for how to manage this long term. I'd rather spend the time defining some clear boundaries for Adyen and Spin to avoid future conflicts that are bound to happen.

Reply:
> yeah this makes me pretty uneasy, I'm not familiar with the structure of this library, or Adyen's willingness to pull in upstream changes, but I would prefer something that we could possibly contribute upstream so that we stay current with their releases.

I don't think they will take our changes. These are spin specific changes we are adding to their UI ie. Spin Terms of service and links to our website.

Reply:
> I think we should have a story for how to manage this long term. I'd rather spend the time defining some clear boundaries for Adyen and Spin to avoid future conflicts that are bound to happen.

I do not expect us to be making too much changes over time. The current changes are just must have UI elements that fits Spin's branding/legal needs. So I would more or less treat our changes like the patches we have for RN libraries. Small set of changes that gets applied to the official versions of the Adyen SDK.

Reply:
If our changes were useful in the broad sense, maybe they would?  Customization of their payment UI is probably something useful to them.

Even if they don't take the changes, I think separating Spin-specific stuff out of this source tree entirely is desirable.

Reply:
Oh yeah, I agree. If we can generalize our UI changes in a way that enables their SDK to be able to handle the customizations we'd want from outside of the SDK and in a way that they would do themselves, then I can definitely see them accepting our changes. However I do think this is a tall order. We were under a gun to deliver Adyen integration with in a sprint and we had to go with the quickest solution.